src/metabase/util/malli/schema.clj

```diff
@@ -267,6 +267,29 @@
      [:value_field {:optional true} Field]
      [:label_field {:optional true} Field]]))
 
+(def RemappedFieldValue
+  "Has two components:
+    1. <value-of-field>          (can be anything)
+    2. <value-of-remapped-field> (must be a string)"
+  [:tuple :any :string])
+
+(def NonRemappedFieldValue
+  "Has one component: <value-of-field>"
+  [:tuple :any])
+
+(def FieldValuesList
+  "Schema for a valid list of values for a field, in contexts where the field can have a remapped field."
+  [:or
+   [:sequential RemappedFieldValue]
+   [:sequential NonRemappedFieldValue]])
```

Comment:
make these private?

Reply:
They shouldn't be private. They are used in other namespaces (well `RemappedFieldValue` isn't, but there's no reason why it could be).